deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
I would expect the admin to provide a label selector indicating what nodes they want fluentd to run on. How were the nodes being targeted before?